### PR TITLE
fix: Several Listing Cleanup Items

### PR DESCRIPTION
**Pull request by aballerr**
https://uniswaplabs.atlassian.net/browse/WEB-1932 - updates banner on view nfts to show different text
https://uniswaplabs.atlassian.net/browse/WEB-1993 - prevents images in bag from stretching
https://uniswaplabs.atlassian.net/browse/WEB-1658 - prevent overflow of text
https://uniswaplabs.atlassian.net/browse/WEB-2008 - hides navbar on mobile for certain pages.

### CODE REVIEW

File: src/nft/components/bag/profile/ListingSection.tsx [hunk 1]
```diff
@@ -110,6 +110,7 @@ export const ListingSection = ({
                       color="textPrimary"
                       textOverflow="ellipsis"
                       overflow="hidden"
+                      whiteSpace="nowrap"
```
Comment (aballerr) [inline src/nft/components/bag/profile/ListingSection.tsx:113]: text is supposed to be ellipsis, it was wrapping.  This fixes

File: src/nft/components/collection/CollectionStats.tsx [hunk 2]
```diff
@@ -275,7 +275,7 @@ const StatsRow = ({ stats, isMobile, ...props }: { stats: GenieCollection; isMob
   const isCollectionStatsLoading = useIsCollectionLoading((state) => state.isCollectionStatsLoading)
 
   // round daily volume & floorPrice to 3 decimals or less
-  const totalVolumeStr = volumeFormatter(stats.stats?.total_volume ?? 0)
+  const totalVolumeStr = volumeFormatter(Number(stats.stats?.total_volume) ?? 0)
```
Comment (aballerr) [inline src/nft/components/collection/CollectionStats.tsx:278]: we are getting a string back here when a number is expected

File: src/nft/components/profile/view/FilterSidebar.tsx [hunk 3]
```diff
@@ -30,7 +50,7 @@ export const FilterSidebar = () => {
       position={{ sm: 'fixed', md: 'sticky' }}
       top={{ sm: '40', md: 'unset' }}
       left={{ sm: '0', md: 'unset' }}
-      width={{ sm: 'full', md: 'auto' }}
+      width={{ sm: 'full', md: '332', lg: '332' }}
```
Comment (aballerr) [inline src/nft/components/profile/view/FilterSidebar.tsx:53]: the search container shrinks when typing text.  This isn't part of my task, just a solution to improve the experience for now

File: src/nft/components/profile/view/FilterSidebar.tsx [hunk 4]
```diff
@@ -99,16 +120,18 @@ const CollectionSelect = ({
...
-              <CollectionItem
-                key={index}
-                collection={collection}
-                collectionFilters={collectionFilters}
-                setCollectionFilters={setCollectionFilters}
-              />
-            ))}
-          </Box>
+          <ItemsContainer>
+            <Box paddingBottom="8" style={{ scrollbarWidth: 'none' }}>
```
Comment (aballerr) [inline src/nft/components/profile/view/FilterSidebar.tsx:124]: this overflowY made a random scrollbar show up when it wasn't needed and when there was no overflow

File: src/nft/components/profile/view/WalletAssetDisplay.tsx [hunk 5]
```diff
@@ -58,8 +58,7 @@ export const WalletAssetDisplay = ({ asset, isSellMode }: { asset: WalletAsset;
   return (
     <Link to={getAssetHref(asset, DetailsOrigin.PROFILE)} style={{ textDecoration: 'none' }}>
       <Column
-        borderBottomLeftRadius="20"
-        borderBottomRightRadius="20"
+        borderRadius="20"
```
Comment (aballerr) [inline src/nft/components/profile/view/WalletAssetDisplay.tsx:61]: hover of assets are square at top, looks really weird

File: src/components/Common/index.tsx [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  scrollbar-color: ${({ theme }) => `${theme.backgroundOutline} transparent`};
+  overflow-y: scroll;
+  height: 100%;
+
+  // safari and chrome scrollbar styling
+  ::-webkit-scrollbar {
+    background: transparent;
+
+    // Set height for horizontal scrolls
+    ${({ isHorizontalScroll }) => {
```
Comment (aballerr) [inline src/components/Common/index.tsx:15]: I reuse this a good amount, not sure where to put it.

Comment (JFrankfurt) [reply src/components/Common/index.tsx:15]: nit: make `isHorizontalScroll` transient https://styled-components.com/docs/api#transient-props

File: src/components/Common/index.tsx [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+      return isHorizontalScroll ? 'height: 4px;' : 'width: 4px;'
```
Comment (JFrankfurt) [inline src/components/Common/index.tsx:16]: nit, make these 
```
css`height: 4px;`
```

File: src/components/NavBar/index.tsx [hunk 8]
```diff
@@ -71,10 +72,25 @@ const PageTabs = () => {
...
+
+  return false
+}
+
 const Navbar = () => {
+  const shouldHideNavbar = useShouldHideNavbar()
   const isNftPage = useIsNftPage()
 
-  return (
+  return shouldHideNavbar ? null : (
```
Comment (JFrankfurt) [inline src/components/NavBar/index.tsx:93]: nit: do an early return instead of a ternary

File: src/nft/components/profile/view/FilterSidebar.tsx [hunk 9]
```diff
@@ -99,16 +105,18 @@ const CollectionSelect = ({
...
+              {displayCollections?.map((collection, index) => (
+                <CollectionItem
+                  key={index}
```
Comment (JFrankfurt) [inline src/nft/components/profile/view/FilterSidebar.tsx:112]: use `collection.address` for the key if possible

File: src/nft/pages/profile/sell.css.ts [hunk 10]
```diff
@@ -22,7 +22,6 @@ export const mobileSellWrapper = style([
     zIndex: { sm: '3', md: 'auto' },
     height: { sm: 'full', md: 'auto' },
     width: 'full',
-    overflowY: 'scroll',
```
Comment (JFrankfurt) [inline src/nft/pages/profile/sell.css.ts:25]: double checking this removal is intentional and handled appropriately

**Review by JFrankfurt — APPROVED** [review]
A bunch of nits--little things I think could be improved here, but none are blocking.

File: src/nft/pages/profile/sell.css.ts (hunk 10 above)
Comment (aballerr) [reply src/nft/pages/profile/sell.css.ts:25]: it's intentional.  There were multiple overflows that were showing a scrollbar when nothing was scrollable and it now only shows when content is actually scrollable

**Comment by aballerr** [comment]
> A bunch of nits--little things I think could be improved here, but none are blocking.

I addressed all of them, all made sense to me

**Pull request merged**